### DISCUSSION TITLE
Option to set less than all entries to read

### DISCUSSION THREAD

**Discussion by tomcat-xx** [tughi/aggregator-android#40] [Ideas]
As discussed on Patreon before: As discussed on Patreon before: Option to set less than all entries to read would be nice, maybe with menu entry "Mark until current position" ore just while scrolling over.

**Comment by Akamar** [comment]
I support this idea; this feature is essential when we have a lot of news. Feeder has options to "mark above as read" and "mark below as read"; you could also try selecting items by date rather than position in the list.
